addons/sale_project/tests/test_project_profitability.py

```diff
@@ -234,13 +240,80 @@ def test_project_profitability(self):
                 },
             },
         )
+        # Create a down payment for a fixed amount of 115
+        context = {
+            'active_model': 'sale.order',
+            'active_ids': [self.sale_order.id],
+            'active_id': self.sale_order.id,
+            'default_journal_id': self.company_data['default_journal_sale'].id,
+        }
+        downpayment = self.env['sale.advance.payment.inv'].with_context(context).create({
+            'advance_payment_method': 'fixed',
+            'fixed_amount': 115,
+            'deposit_account_id': self.company_data['default_account_revenue'].id
+        })
+        # the down payment is with the amount with the default 15% included, therefore, the amount of the dp is higher than the amount of the sol created.
```

Comment:
While we are at it, please standardize your comment, capital-letter and punctuation wise :pray: